package.json

```diff
@@ -667,6 +667,12 @@
             "default": false,
             "description": "%configuration.maven.pomfile.prefetchEffectivePom%",
             "scope": "application"
+          },
+          "maven.warnConflictsInDiagnostics": {
```

Comment:
I searched "diagnostics" in existing settings, TS is using 
 "tsserver.enableProjectDiagnostics". So maybe something like "maven.dependency.enableConflictDiagnostics` would be better? Please comment on name of this setting.  @jdneo

Reply:
+1 for maven.dependency.enableConflictDiagnostics

Reply:
@melody618 let's use this name